manage.py

```diff
@@ -312,3 +314,17 @@ def config(config_key):
 @app.cli.command()
 def set_celery_beat_healthy():
     return celery_beat_health_check()
+
+
+@app.cli.command()
+def healthcheck():
+    """Calls the healthcheck API"""
+    result = requests.get(
+        url_for('check')
+    )
+    print(result.text)
+
+    if not result.ok:
+        return result.status_code
```

Comment:
Did this work? I thought you had to the below, but not sure if flask adds some magic:
```suggestion
        sys.exit(result.status_code)
```

Reply:
I simplified this a little bit. Let me know if what I did works.